reactor-core/src/main/java/reactor/core/publisher/SerializedSubscriber.java

```diff
@@ -232,6 +259,25 @@ else if (d) {
 		}
 	}
 
+	private void discardMultiple(LinkedArrayNode<T> head) {
+		synchronized (actual) {
+		LinkedArrayNode<T> originalHead = head;
```

Comment:
nit: wrong indentation

Reply:
ah snap that is actually a leftover that wasn't supposed to be here